### ISSUE TITLE
Verification Checker

### ISSUE THREAD

**Issue by garyluu** [dockstore/dockstore#1954]
### Desired behaviour
Should have a documented section explaining how the new verification works.
- Are people supposed to verify the checker test.json or the original entry's test.json
- Does verifying checker's test.json also cause the original entry to be verified
- What happens with entries that do not have checkers and were previously verified or about to be verified
- Does verifying the original entry also cause the checker to be verified?

┆Issue is synchronized with this [Jira Story](https://ucsc-cgl.atlassian.net/browse/DOCK-288)
┆Fix Versions: Dockstore 1.11
┆Issue Number: DOCK-288
┆Sprint: Sprint 63- Nurse Shark
┆Issue Type: Story

**Comment by denis-yuen** [comment]
1. Agree, the endpoint should verify the actual json that was tested even though it is not a json for the underlying workflow
2. Yes, end-users will only see (without digging) the underlying workflow, so that is what what will show the verification information (will probably need to rethink how things are displayed without changing the database)
3. They will be verified, but it should just not show that they were verified via checker workflow
4. In database, no. But keep in mind #1933

**Comment by coverbeck** [comment]
>  explaining how the new verification works

By "new verification", do you mean checker workflows?

> Probably need to chat with Gary and I based on how we want to position this for B/S/G

Reading this part of the doc for the first time, there's definitely an overlap, in particular the "[recommended characteristics](https://docs.dockstore.org/en/develop/advanced-topics/verification.html?highlight=verification)"

**Comment by denis-yuen** [comment]
> By "new verification", do you mean checker workflows?

At the time, this meant a rework of verification in PRs like https://github.com/dockstore/dockstore/pull/2844
or issues like https://github.com/dockstore/dockstore/issues/2384

**Comment by coverbeck** [comment]
Piecing things together for myself, the [public documentation](https://docs.dockstore.org/en/develop/advanced-topics/verification.html?highlight=verification#additional-verification-information) cited by Denis in the previous comment was added a year after this ticket was written in this [commit](https://github.com/dockstore/dockstore-documentation/commit/ee928e4155e4f997b1d30e43df1a2d3b987863b0). It addresses "Should have a documented section explaining how the new verification works."

Regarding the other questions, it looks like checkers and their workflows are separate entities in terms of verification. Verifying one has no bearing on verification of the related entry. I think we should keep it like that. It's going to be tricky otherwise if updating the verification of a source file in one also updates the other (what if the source file doesn't exist in the other, etc.?). It does potentially lead to some weird scenarios, e.g., where the checker is verified but the workflow is not, but I think it's OK.

That leaves the b/s/g which I think we should deal with as that definition fills out.

Bottom line proposal: close this ticket with no more work.

**Comment by coverbeck** [comment]
Closing, see previous comment.
